### PR TITLE
PLAT-126339: [Agate] Added UI tests for RangePicker

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - Added ui tests for RangePicker

### Links
[//]: # (Related issues, references)
PLAT-126339

### Comments

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
:shipit:

File: tests/ui/specs/RangePicker/RangePicker-specs.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		beforeEach(function () {
+			Page.open();
+		});
+
+		describe('default', function () {
+			const rangePicker = Page.components.rangePickerDefault;
+
+			describe('5-way', function () {
+				it('should change the value forward when incrementing the rangePicker', function () {
+					browser.waitUntil(() => rangePicker.decrementer(rangePicker.self).isFocused(), {timeout: 1500,  interval: 100});
```
Comment (MikyungKim) [inline tests/ui/specs/RangePicker/RangePicker-specs.js:16]: Also, here please check these `wailtUntil` are mandatory or not.

Comment (daniel-stoian-lgp) [reply tests/ui/specs/RangePicker/RangePicker-specs.js:16]: already on it 👍

File: tests/ui/specs/RangePicker/RangePickerPage.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+'use strict';
+const {Page} = require('@enact/ui-test-utils/utils');
+const {element} = require('@enact/ui-test-utils/utils');
+
+class AgateRangePickerInterface {
```
Comment (MikyungKim) [inline tests/ui/specs/RangePicker/RangePickerPage.js:5]: Please remove `Agate` here.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**